contrib/http_util/src/test/java/io/opencensus/contrib/http/FakeHttpResponse.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.contrib.http;
+
+/** Fake HTTP response entity class. */
+public abstract class FakeHttpResponse {
```

Comment:
If you want this to be public consider to put it in a package:
package io.opencensus.contrib.http.testing